beetmoverscript/task.py

```diff
@@ -148,31 +132,12 @@ def get_release_props(context, platform_mapping=STAGE_PLATFORM_MAP):
     """determined via parsing the Nightly build job's balrog_props.json and
     expanded the properties with props beetmover knows about."""
     payload_properties = context.task.get('payload', {}).get('releaseProperties', None)
-    initial_release_props_file = None
-
-    if payload_properties:
-        props = payload_properties
-        log.debug("Loading release_props from task's payload: {}".format(props))
-    else:
-        initial_release_props_file = get_initial_release_props_file(context)
-        props = utils.load_json(initial_release_props_file)['properties']
-        log.warn(
-            'Deprecated behavior! This will be gone after Firefox 59 reaches release. Loading release_props from "{}": {}'
-            .format(initial_release_props_file, props)
-        )
-
-    final_props = update_props(context, props, platform_mapping)
-    return (final_props, initial_release_props_file)
 
+    # if payload_properties:
+    props = payload_properties
```

Comment:
This looks good; however, I think we can make two small improvements:

1. Since we're dropping the balrog_props behavior, it means the release_properties field in `task.payload` is mandatory. So I think it's worth barfing if those values are not set. 
2. I don't really see the value of `props` here, since it's just another reference to `payload_properties`.

That being said, I'd rewrite the function to be something like:
```
def get_release_props(context, platform_mapping=STAGE_PLATFORM_MAP):
    """Function to grab the release properties data from task payload"""
    payload_properties = context.task.get('payload', {}).get('releaseProperties', None)
    if not payload_properties:
        raise ScriptWorkerTaskException(
            "could not determine release props file from task payload"
        )

    log.debug("Loading release_props from task's payload: {}".format(payload_properties))
    return update_props(context, payload_properties, platform_mapping)

```